RubberduckTests/Inspections/ImplicitActiveWorkbookReferenceInspectionTests.cs

```diff
@@ -11,7 +11,6 @@ namespace RubberduckTests.Inspections
     public class ImplicitActiveWorkbookReferenceInspectionTests
     {
         [Test]
-        [Ignore("This was apparently only passing due to the test setup. See #4404")]
```

Comment:
Are we sure we want to unignore this? The comment suggests it is a false positive?

Reply:
@bclothier I'm missing something. The test setup puts this code in a standard module, where an unqualified `Worksheets` call is indeed an implicit `ActiveWorkbook` reference. Not sure what's wrong with this test.

Reply:
It refers to a closed issue.

Reply:
@retailcoder Neither can I see what is wrong. The comment only say test setup is flawed but it doesn't say how. I do see that issue was closed but when reviewing [the git blame](https://github.com/rubberduck-vba/Rubberduck/blame/master/RubberduckTests/Inspections/ImplicitActiveWorkbookReferenceInspectionTests.cs), this came from #4425 / #4414, and [was only one line of change](https://github.com/rubberduck-vba/Rubberduck/pull/4425/files#diff-3192feb08266c2ee9c7904879905274c). The referenced issue #4404 (and its related cousin) is indeed closed but so far there has been no changes in the test setup. I would have had expected there to be a change in the test setup to warrant the removal of the attribute. 

Because it's without an explanation, I'm in favor of not ignoring this until we actual have an example of why the test setup is invalid.

Reply:
The test setup has been fixed in commit f28ed0d1 (use serialized COM projects instead of serialized declarations), which is also the commit that introduced the ignore.

When the issue in the inspection got fixed,, we simply missed to unignore the test.